config/resources/domain.lisp

```diff
@@ -3,4 +3,6 @@
 
 (defparameter *include-count-in-paginated-responses* t)
 ;(defparameter *cache-count-queries-p* t)
-;(defparameter *supply-cache-headers-p* t)
\ No newline at end of file
+;(defparameter *supply-cache-headers-p* t)
+
+(defparameter *max-group-sorted-properties* nil)
```

Comment:
Maybe add a comment why this was needed? We will probably thank ourselves in the future :smile: